### PR TITLE
Ajout article de blog Une application React plus réactive - Le lazy load

### PR DESCRIPTION
**Pull request by frisoty**

### CODE REVIEW

**Comment by ElevenWilson** [comment]
20 words might be misspelled, please check them: baseurl, CSS, gif, google, img, IntersectionObserver, jpeg, jpg, JS, kcordier, l’optimistic, minitel, m'vois, onLoad, référencement, SEO, squoosh, ui, useIntersectionObserver, UX

**Comment by ElevenWilson** [comment]
20 words might be misspelled, please check them: baseurl, CSS, gif, google, img, IntersectionObserver, jpeg, jpg, JS, kcordier, l’optimistic, minitel, m'vois, onLoad, référencement, SEO, squoosh, ui, useIntersectionObserver, UX

File: _posts/fr/2020-01-22-lazy-load-react.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+Bon ! Vous avez fluidifier les interactions de votre application grâce à [l’optimistic ui](https://blog.eleven-labs.com/fr/optimistic-ui-avec-react-et-apollo-js/), vous avez diminué l'impression d’attente lors du chargement de vos pages grâce au [skeleton screen](https://blog.eleven-labs.com/fr/skeleton-screen-avec-react-et-apollo-js/) et malgré tout ça vos pages mettent encore trop longtemps à s'afficher. J’ai une petite question pour vous :
+
+Qu’est ce qui fait en moyenne 50% du poids de votre page (selon [HTTP Archive](https://httparchive.org/)) et qui serait susceptible de ralentir le chargement de notre site ?
+.
+.
+.
+C’est élémentaire mon cher Wilson… C’est les **images**.
+
+Donc la solution ici est simple, réduire le poids des images de votre page. Pour cela, il existe une multitude de services en ligne, comme par exemple l’outil de google [squoosh.app](http://squoosh.app). Mais on peut aller encore plus loin.
```
Comment (VEBERArnaud) [inline _posts/fr/2020-01-22-lazy-load-react.md]:
```suggestion
Donc la solution ici est simple, réduire le poids des images de votre page. Pour cela, il existe une multitude de services en ligne, comme par exemple l'outil de google [squoosh.app](http://squoosh.app). Mais on peut aller encore plus loin.
```

Comment (VEBERArnaud) [reply _posts/fr/2020-01-22-lazy-load-react.md]: j'ai l'impression que ca pose problème au moment du build
https://travis-ci.org/eleven-labs/blog.eleven-labs.com/builds/640142104#L413

**Comment by ElevenWilson** [comment]
20 words might be misspelled, please check them: baseurl, CSS, gif, google, img, IntersectionObserver, jpeg, jpg, JS, kcordier, l’optimistic, minitel, m'vois, onLoad, référencement, SEO, squoosh, ui, useIntersectionObserver, UX

**Comment by ElevenWilson** [comment]
21 words might be misspelled, please check them: baseurl, CSS, gif, google, img, interchanger, IntersectionObserver, jpeg, jpg, JS, kcordier, l’optimistic, minitel, m'vois, onLoad, référencement, SEO, squoosh, ui, useIntersectionObserver, UX

**Comment by ElevenWilson** [comment]
21 words might be misspelled, please check them: baseurl, CSS, gif, google, img, interchanger, IntersectionObserver, jpeg, jpg, JS, kcordier, l’optimistic, minitel, m'vois, onLoad, référencement, SEO, squoosh, ui, useIntersectionObserver, UX

**Pull request merged**